values.yaml

```diff
@@ -1,402 +1,134 @@
-  # pods.
-  # To unset a value, set it to "".
-  # To unset all values, you can provide a values.yaml file which sets resources
-  # to nil. See values.yaml for an example.
-  #
-  # e.g:
-  # envproxy:
-  #   # This will cause all values to be unset.
-  #   resources:
-  #   replica: 1
+  # coderd.resources -- Kubernetes resource specification for coderd pods.
+  # To unset a value, set it to "". To unset all values, set resources to nil.
   resources:
     requests:
       cpu: "250m"
       memory: "512Mi"
     limits:
       cpu: "250m"
       memory: "512Mi"
-  # envproxy.terminationGracePeriodSeconds -- Amount of seconds to wait before
-  # shutting down the environment proxy if there are still open connections.
-  # This is set very long intentionally so developers do not deal with
-  # disconnects during deployments.
-  terminationGracePeriodSeconds: 14400
-
-# timescale -- Contains configuration for the internal database. We don't
-# recommend running this service in production. See the `postgres` section for
-# connecting to an external Postgres database.
-timescale:
-  # timescale.image -- Injected during releases.
-  image: ""
-  # timescale.resources -- Kubernetes resource request and limits for the
-  # timescale pod.
-  # To unset a value, set it to "".
-  # To unset all values, you can provide a values.yaml file which sets resources
-  # to nil. See values.yaml for an example.
-  #
-  # e.g:
-  # timescale:
-  #   # This will cause all values to be unset.
-  #   resources:
-  resources:
-    requests:
-      cpu: "250m"
-      memory: "1Gi"
-      # timescale.resources.requests.storage -- Specifies the size of the
-      # volume claim for persisting the database.
-      storage: "10Gi"
-    limits:
-      cpu: "250m"
-      memory: "1Gi"
 
+# envbox -- Required for running Docker inside containers. See requirements:
+# https://coder.com/docs/coder/v1.19/admin/workspace-management/cvms
 envbox:
-  # envbox.image -- Injected during releases.
+  # envbox.image -- Injected by Coder during release.
   image: ""
 
-# environments defines configuration that is applied to all
-# user environments.
-environments:
-  # environments.tolerations -- Tolerations are applied to all user
-  # environments. Each element is a regular pod toleration object. To set
-  # service tolerations see serviceTolerations. See values.yaml for an example.
-  #
-  # e.g.
-  # tolerations:
-  # - key: "my key"
-  #   operator: "Exists"
-  #   effect: "NoExecute"
-  #   tolerationSeconds: 6000
-  tolerations: []
-
-  # environments.nodeSelector -- nodeSelector is applied to all user
-  # environments to specify eligible nodes for environments to run on.
-  # See: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector
-  # 
-  # eg.
-  # nodeSelector:
-  #   disktype: ssd
-  nodeSelector: {}
-
-# certs -- Describes CAs that should be added to Coder services. These certs
-# are NOT added to environments.
-certs:
-  secret:
```

Comment:
I think it's useful to have a way to inject certs into our pods; we should expand the feature rather than deprecating it, IMO. The new TLS support is great for handling server certs (public/private keys) but that doesn't cover outbound connections from Coder to other services:

1. CA certificates that Coder uses to validate the servers to which it connects
2. Client certificates that Coder presents to remote servers

Reply:
Could you write up the syntax of what you imagine? I'm happy to implement, just not as familiar with naming conventions in that realm.

Reply:
just leave these out for now - from the directory structure, it looks like the `certs/` directory was meant to hold cert files, which we could load into Secrets using https://helm.sh/docs/chart_template_guide/accessing_files/#configmap-and-secrets-utility-functions

so I think having a way to specify filenames would be convenient, as well as to handle all the different types of certs that people would want to configure, as defined in the RFC: https://www.notion.so/TLS-certificates-e17259ebcaf84f64969ba6a9351d38a7

Reply:
Going to leave this in for right now and just move it. No need for action on your part!